### DISCUSSION TITLE
Settings entries reorganization to remove duplicates and make it more intutive

### DISCUSSION THREAD

**Discussion by Wladefant** [Docile-Alligator/Infinity-For-Reddit#1080] [Ideas]
The same discussion on [Reddit](https://www.reddit.com/r/Infinity_For_Reddit/comments/xaunl0/what_are_your_ideas_for_the_settings/?utm_source=share&utm_medium=web2x&context=3)

I think the settings need to be restructured as they are very counterintuitive and contain duplicates:

My proposal would be something like (not complete):

**If you have suggestions you can add a comment with the suggestion down below**

Settings:
Search bar 
<img src="https://user-images.githubusercontent.com/79611093/186868809-2886a4ba-27ac-4279-9356-30088f57d948.jpg" width=50% height=50%>

- Appearance
  - Main Theme
    - Select Theme (dropdown)
    - Amoled Dark (toggle)
    **Material You**
    - Enable Material You
    - Light Theme
    - Dark Theme
    - Amoled Theme
    - Manage Themes
  - Post Layout
    - Seperate Post and Comments in Landscape Mode
    - Hide Post Type
    - Hide Post Flair
    - Hide Upvote Ratio
    - Hide the Number of Awards
    - Hide Subreddit and User Prefix
    - Hide the Number of Votes
    - Hide the Number of Comments
  - Comment Layout
    - Show Top-level Comments First
    - Show Comment Divider
    - Show Only One Comment Level Indicator
    - Comment Toolbar Hidden by Default
    - Click to Show/Hide Comment Toolbar
    - Fully Collapse Comment
    - Show Author Avatar
    - Hide Comment Awards
    - Always Show the Number of Child Comments
    - Show Fewer Toolbar Options Starting From ...
  - Video Layout
    - Legacy Video Controller UI
    - Easier to Watch in Full Screen
    - Autoplay Videos Visible Area Offset (Portrait) (Slider)
    - Autoplay Videos Visible Area Offset (Landscape) (Slider)

- Network
  **Data Saving Mode** (Title)
  - Data Saving Mode (dropdown)
  - Disable Image Preview
  - Only Disable Preview in Video and Gif Posts

- Security
  - Require Authentication to Show Accounts
  - Secure Mode
  - App Lock
  - App Lock Timeout

- Notification
  - Enable Notifications (toggle)
  - Check Notifications Interval
  - Show Device Notification Settings

- Storage
  - Image Download Location
  - Gif Download Location
  - Video Download Location
  - Separate Folder for Each Subreddit
  **NSFW** (Title)
  - Save NSFW Media in Different Location
  - NSFW Download Location

  - Import/Export
     **Database** (Title)
    - Delete All Subreddits
    - Delete All Users
    - Delete All Sort Types 
    - Delete All Post Layouts
    - Delete All Themes
    - Delete All Front Page Scrolled Positions
    - Delete All Read Posts

     **Settings** (Title)
    - Delete All Legacy Settings
    - Reset All Settings
    - Backup Settings
    - Restore Settings

 ---
**Project** (Title)
- Documentation (Link to Github wiki)
- Forum (subtitle: support, questions, feature requests) (Link to github discussions)
- Contribute (Link to contribution guidelines)
- Report Bug (Link to github issues)
- About
  - Infinity For Reddit version (subtitle: 2.6.1)
  - Source Code (Link to github project)
  - Rate on Google Play
  - Subreddit 
  - Opensource libraries (instead of Acknowledgement)
  - Credits
  - View Changelog (alt. Name: what's new)
  - Privacy Policy 
  - Reddit User Agreement 

---

Alternative Settings Around reddit Elements

- Post
- Comment
- Media
  - Video
  - Gif
  - Image
  - Link

- Theme
  - Select Theme (dropdown)
  - Amoled Dark (toggle)
  **Customiazation**
  - Enable Material You
  - Light Theme
   - customize Theme...
  - Dark Theme
   - customize Theme...
  - Amoled Theme
   - customize Theme...
  - Manage Themes
  

- Security
  - Require Authentication to Show Accounts
  - Secure Mode
  - App Lock
  - App Lock Timeout

- Notification
  - Enable Notifications (toggle)
  - Check Notifications Interval
  - Show Device Notification Settings

 ---
**Project** (Title)
- Documentation (Link to Github wiki)
- Forum (subtitle: support, questions, feature requests) (Link to github discussions)
- Contribute (Link to contribution guidelines)
- Report Bug (Link to github issues)
- About
  - Infinity For Reddit version (subtitle: 2.6.1)
  - Source Code (Link to github project)
  - Rate on Google Play
  - Subreddit 
  - Opensource libraries (instead of Acknowledgement)
  - Credits
  - View Changelog (alt. Name: what's new)
  - Privacy Policy 
  - Reddit User Agreement

**Comment by Dragonfly-30** [comment]
I think it'll be better to add a toggle or something to comment indicator to turn it grey in comment settings(ik about that in themes section) and if possible then to an option to adjust padding.

**Comment by jcperil** [comment]
Definitely yes for the search bar. Let's be honest, Infinity provides a lot of options in settings (which I mean in a good way).
